src/index.js

```diff
@@ -0,0 +1,153 @@
+    div.innerHTML += `
+    <div class="movie" id="${movie.id}">
+    <img src="${movie.image.medium}" alt="${movie.name}">
+    <div class="likes">
+    <p class="like-p">${movie.name}</p>
+    <button class="like-button"><i class="far fa-heart fa-2x" id='heart-${movie.id}'></i></button>
+    </div>
+    <p>${likeVal} likes</p>
+    <button class="comment-button">Comment</button>
+    </div>`;
+  });
+};
+
+const updateLikes = async (ele) => {
+  await axios.post('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/XMHWey4za3iNnBFD5KUq/likes', { item_id: ele.id });
+  displayMovie();
+};
+
+const popUpHtml = (target) => {
+  comments.innerHTML = `
+    <div id='pop'>
+      <div id='img-div'>
+        <i class='fas fa-times'></i>
+        <img src="${movies[parseInt(target, 10) - 1].image.medium}" alt="${movies[parseInt(target, 10) - 1].name}">
+      </div>
+      <h2>${movies[parseInt(target, 10) - 1].name}</h2>
+      <div id='comment-feature'>
+        <p> Rating: ${movies[parseInt(target, 10) - 1].rating.average}</p>
+        <p> Genres: ${movies[parseInt(target, 10) - 1].genres}</p>
+        <p> Released Date: ${movies[parseInt(target, 10) - 1].premiered}</p>
+        <p> Language: ${movies[parseInt(target, 10) - 1].language}</p>
+      </div>
+    </div>`;
+};
+
+const getComment = async (movieId) => {
+  try {
+    const comments = await axios.get(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/XMHWey4za3iNnBFD5KUq/comments?item_id=${movieId}`);
+    return comments.data;
+  } catch {
+    return [];
+  }
+};
+const showComments = async (id) => {
+  try { getCommentFromAPI = await getComment(id); } catch {
+    popUpHtml(id);
+    commentHTML = '<p>No Comments Yet</p>';
+    getCommentFromAPI = [];
+  }
+  if (getCommentFromAPI.length === 0) {
+    commentHTML = '<p>No Comments Yet</p>';
+  } else {
+    getCommentFromAPI.forEach((i) => { commentHTML += `<p>${i.creation_date} ${i.username}: ${i.comment}</p>`; });
+  }
+  const commentCount = itemsCounter(getCommentFromAPI);
+  comments.innerHTML
+  += `<div id='comment-area'>
+        <h2>Comments (${commentCount})</h2>
+        ${commentHTML}
+      </div>
+      <div id='comment-form'>
+        <h2>Add a Comment</h2>
+        <form action="submit" id="form-area">
+          <input type="text" id="name" placeholder="Your Name" required>
+          <textarea type="textarea" rows="4" cols="50" name="comment" placeholder="Your Insights" required></textarea>
+          <p></p>
+          <button name="${id}" type="button">Submit Comment</button>
+        </form>
+      </div>`;
+};
+
+const postNewComments = (movieID, userName, userComment) => {
+  axios.post('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/XMHWey4za3iNnBFD5KUq/comments', {
+    item_id: movieID,
+    username: userName,
+    comment: userComment,
+  });
+  showComments(movieID);
+};
```

Comment:
- [ ] [OPTIONAL] It is expected according to the project requirements that when you submit a new comment, the data gets POSTED to the involvement API and the screen gets updated. Please see below 👇 
![image](https://user-images.githubusercontent.com/46839548/205977262-4838e57f-6aec-4b0f-8c7e-cdc7906905d2.png)